scripts/avail-ext-doc/README.md

```diff
@@ -2,16 +2,34 @@
 
 Automatically create PR to update the document listing the available CLI extensions.
 
+To build locally:
+
 ```bash
-docker build . -t derekbekoe/az-avail-ext-doc-sync:0.1.2
+docker build . -t sttramer/az-ext-list-publisher:0.2.0
```

Comment:
Should the image name be __user__ agnostic? Instead of fill in the author's name is there any other alternative?

Reply:
I would absolutely prefer to be publishing this to a Microsoft-owned Hub repo, or a team-owned ACR, rather than to my private repository. If you can help me figure out the correct credentials and way to configure Travis to do so as part of this PR, I'm all for it.